CONTRIBUTING.md

```diff
@@ -50,8 +50,21 @@ Before you can run tests, PHPUnit must be installed. You can install it using Co
 composer install
 ```
 
+### Format and test the code
+
+Run the following command to run prettier on the code, to format the code:
+```
+composer run format
+```
+
+Run the following commad to check if your code is formatted properly:
+```
+composer run format:check
+```
+NOTE: You need to have [`prettier`](https://prettier.io/) and the [prettier-php plugin](https://github.com/prettier/plugin-php) installed globally in order to run this command.
```

Comment:
For notes, we can use the GitHub [note highlight](https://github.com/community/community/discussions/16925) feature (in beta)

> **Note** You need to have [`prettier`](https://prettier.io/) and the [prettier-php plugin](https://github.com/prettier/plugin-php) installed globally in order to run this command.

```suggestion
> **Note** You need to have [`prettier`](https://prettier.io/) and the [prettier-php plugin](https://github.com/prettier/plugin-php) installed globally in order to run this command.
```